routers/wishlist-router.js

```diff
@@ -0,0 +1,109 @@
+const express = require("express");
+const router = express.Router();
+const mongoose = require("mongoose");
+const lodash = require("lodash")
+const {extend} = lodash
+
+const { Wishlist } = require("../models/wishlist-model")
+
+router.route("/")
+.get(async (req, res) => {
+    try {
+        const wishlist = await Wishlist.find({})
+        res.status(200).send(wishlist)
+    } catch (err) {
+        res.status(404).json({success: false, message: "could not load wishlist", errMessage: err.message})
+    }    
+})
+.post(async (req, res) => {
+    try {
+        const wishlist = req.body
+        const newWishlist = new Wishlist(wishlist)
+        const savedWishlist = await newWishlist.save()
+        res.status(200).json({success: true, newWishlist: savedWishlist})
+    } catch (err) {
+        res.status(404).json({success: false, message: "save wishlist failed", errMessage: err.message})
```

Comment:
404 may not be the write status code to send in this situation.

Reply:
will check and add the right code, thanks!